### ISSUE TITLE
Browserosaurus does not show when opening downloaded html file

### ISSUE THREAD

**Issue by aaronkollasch** [will-stone/browserosaurus#564]
### Is there an existing issue for this?

- [X] I have searched the existing issues
- [X] I have searched the [discussions forum](https://github.com/will-stone/browserosaurus/discussions)
- [X] I understand this form is for reporting bugs, and not for requesting [support for new browsers or apps](https://github.com/will-stone/browserosaurus/blob/master/guide/supporting-a-browser-or-app.md).

### Current Behaviour

When opening a downloaded html file, Browserosaurus grabs focus but no window appears.

### Expected Behaviour

Browserosaurus shows the browser picker, a prompt to approve opening the file, or some error message stating that the file could not be opened.

### Steps To Reproduce

1. Download an html file from the web, e.g. https://example.com/index.html
2. Right-click the file -> Open With -> Browserosaurus (or double-click if Browserosaurus is the default app for opening html files)

### Browserosaurus version

19.0.0

### macOS version

12.5

### CPU Architecture

ARM

### Anything else?

The downloaded file has the `com.apple.quarantine` attribute, and removing it with `xattr -d com.apple.quarantine index.html` allows the browser picker to appear as expected.

**Comment by will-stone** [comment]
Thanks. I've not heard of `com.apple.quarantine`, I'll have to do some reading up on that.

**Comment by will-stone** [comment]
I don't really know the solution here as Browserosaurus cannot see the incoming URL (file) 🤷‍♂️

**Comment by mreid-tt** [comment]
> I don't really know the solution here as Browserosaurus cannot see the incoming URL (file) 🤷‍♂️

Hmm, I'm sure there must be a way to handle this since other apps can open the file directly. Consider these command prompt example:

```
% open -a "Brave Browser" index.html
% open -a "Safari" index.html
```

But if we use 'B', it fails if you use the command:

```
% open -a "Browserosaurus" index.html
```

So I'm thinking there must be some way for 'B' to be configured to accept input using the standard MacOS open command (from `/usr/bin/open`).

**Comment by will-stone** [comment]
This should now be fixed in v20.9.1 but let me know if you still have issues with it.

**Comment by mreid-tt** [comment]
> This should now be fixed in v20.9.1 but let me know if you still have issues with it.

I've just upgraded and I don't see any changed behaviour based on the original steps to reproduce.

**Comment by mreid-tt** [comment]
> Strange, seems to be okay for me. Your command works with B too.

Strange indeed. Neither works for me...

https://github.com/user-attachments/assets/c3360b9f-07fa-49df-adea-c2a019b323ae

**Comment by will-stone** [comment]
You're right; just tried it on my work machine and it still doesn't work. Boo 😭

**Comment by will-stone** [comment]
@AlexStrNik thanks. Could you please test and PR that. I can see it says “public.html”. Is that a special string? Otherwise won’t it only work for files called “public”?

**Comment by AlexStrNik** [comment]
I'm sorry, maybe next week I will have more free time. "public.html" is a special string, I'm not sure why it is like this, but it works for me.
